Makefile

```diff
@@ -1,31 +1,34 @@
 # Makefile for Truss.
 #
-SHA := $(shell git rev-parse --short=10 HEAD)
 
-MAKEFILE_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-VERSION_DATE := $(shell $(MAKEFILE_PATH)/commit_date.sh)
+# Pre-install required binaries
+$(shell go get github.com/golang/protobuf/protoc-gen-go)
```

Comment:
Do not do this. This means that whenever the make file is run these will be executed. Even when running tests.